blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/EventElasticSearchIO.java

```diff
@@ -54,49 +58,89 @@ public EventElasticSearchIO(ElasticClientManager manager) {
     }
 
     @Override
-    public void insert(String tenant, List<Map<String, Object>> events) throws Exception {
+    public void insert(String tenantId, Map<String, Object> event) throws Exception {
         final Timer.Context eventInsertTimerContext = eventInsertTimer.time();
-        BulkRequestBuilder bulk = client.prepareBulk();
-
-        for (Map<String, Object> event : events) {
-            event.put(Event.FieldLabels.tenantId.toString(), tenant);
-            IndexRequestBuilder requestBuilder = client.prepareIndex(EVENT_INDEX, ES_TYPE)
-                    .setSource(event)
-                    .setRouting(tenant);
-            bulk.add(requestBuilder);
-        }
-        bulk.execute().actionGet();
+        event.put(Event.FieldLabels.tenantId.toString(), tenantId);
+
+        elasticsearchRestHelper.indexEvent(event);
         eventInsertTimerContext.stop();
     }
 
+//    @Override
```

Comment:
You could delete this commented out code.

Reply:
yes please, all the commented out code should be deleted:
1. it makes the diffs easier to read, (that way the old code is diffed against the new code, instead of both appearing.)
2. we don't want the code to be littered with multiple old versions.

Reply:
yes. I agree. I deleted all of the commented code.